### DISCUSSION TITLE
Generating Gerber and Drill Files for the PCB

### DISCUSSION THREAD

**Discussion by cptbucky** [Schluggi/AIOsense#26] [Q&A] [answered]
Hi all, first of all this may not be necessary and may just be for my own learning. 

I was going to submit a PR for the V1 board to correct the motion sensor pins, however when it comes to the generation of the Gerber files I can see that there's a bit of a difference between my own generation and that in the repository already. 

The following are the differences I'm seeing and so I expect I'm not selecting the right options as I'm using defaults and have never done this before!

- 2 DRL files for NPTH and PTH (and GBR files for each) vs a single DRL file
- I have no CSV file generated which I suspect relates to the positioning and content of labels

Apologies if this type of discussion is not the correct format, willing to move it to wherever appropriate. Thanks in advance!

**Comment by Schluggi** [comment, marked as answer]
Hi @cptbucky,

sorry for the delay and thank you for your afford. I would recommend to not to do anything but ignoring the V1. Just buy a V2 board :)

But to answer your points...
You can check a box to merge both drill files together. This was default I think. If you have any reasons to separate these files I would change this for the next release.
![image](https://user-images.githubusercontent.com/43608073/221976255-5754551c-9ef9-4f8f-8585-c41cd02d9be7.png)

You have to generate the position file separately. I will have this in mind for the final release. 
![image](https://user-images.githubusercontent.com/43608073/221977394-a872f251-66ff-47d7-84c5-ba22512e24c1.png)

**Reply by cptbucky** [reply]
This is amazing, thank you @Schluggi
